cdc/kv/client.go

```diff
@@ -543,6 +537,15 @@ func (c *CDCClient) singleEventFeed(
 	}
 }
 
+func updateCheckpointTS(checkpointTs *uint64, newValue uint64) {
+	for {
```

Comment:
no need to use the for loop?

Reply:
yes, we need

we need retry if `CompareAndSwap` is failed.

Reply:
right, found the `checkpointTs` can be updated by both background sorter and when receiving resolvedTs